content/posts/easystats_threeyears.Rmd

```diff
@@ -114,11 +114,11 @@ If you also want to make your laptops pretties, [get in touch](https://twitter.c
 
 ### Future plans
 
-This project is now approaching a stage where we can start thinking about finally publishing the meta-package *easystats* on CRAN, which will give users a single entry point to access the entire ecosystem's functionality.
+This project is now approaching a stage where we can start thinking about finally publishing the meta-package *easystats* on CRAN, which will give users a single entry point to access the entire ecosystem's functionality. It won't be easy though, as CRAN typically doesn't like this type of meta packages, but we have already some ideas to make it work.
```

Comment:
I think we might get pushback on this; there are MANY meta-packages on CRAN, and CRAN doesn't have any problem with it.

We just haven't tried publishing it to CRAN recently. If we do, it's highly likely to be accepted.

Reply:
true true i'll remove

Reply:
it prolly was just my personal anxiety speaking haha